.github/workflows/main-build.yml

```diff
@@ -110,4 +110,19 @@ jobs:
         config_mod: .functionality.version := 'main_build'
         platform: docker
         src: ${{ matrix.component.dir }}
-        setup: push
\ No newline at end of file
+        setup: push
+
+    # Retag docker images and push to Viash-Hub
+    - name: Login to Viash-Hub container registry
+      uses: docker/login-action@v2
+      with:
+        registry: viash-hub.com:5050
+        username: ${{ secrets.VIASHHUB_USER }}
+        password: ${{ secrets.VIASHHUB_PAT }}
+
+    - name: Retag image
+      run: |
+        componentName=${{ matrix.component.name }}
+        imageName=${componentName/\//_}
```

Comment:
This feels like a dangerous assumption to make. N.B. The default namespace separator is now `/` instead of `_`, but `_` was kept for now. 

At least we should add a comment here on why we're changing the / to _.

@DriesSchaumont Should we change the default namespace separator at some point?

Reply:
Yes, I have made this mistake before. The workflows should work regardless of the chosen namespace separator. I solved this in the integration workflow by adding the separator to the name of the component in the namespace: https://github.com/openpipelines-bio/openpipeline/blob/46c62523a46c5214faee7952ec9fcf0007140a6e/.github/workflows/integration-test.yml#L64

Changing the default namespace separator is a different discussion IMO because, like I said, the workflow should work regarless of the chosen namespace separator.

Reply:
Good point and thanks for the snippet!
Currently no longer relevant as we're currently considering rebuilding the images, but if we circle back to retagging, then it's certainly something to pick up again.